data_parser.py

```diff
@@ -336,6 +336,45 @@ def _parse(cls, stream: BytesIO, ctx: OrderedDict):
     def _build(cls, obj, ctx: OrderedDotDict):
         return StarByteArray.build(obj.encode("utf-8"), ctx)
 
+class StarJson(Struct):
```

Comment:
If I'm remembering my StarryPy right, this is the same as the Variant data type further down the file. We wrote this before we had the real names for these data structures, so it can be a bit confusing.

Reply:
Removed this duplicate type